api/src/pcapi/core/users/api.py

```diff
@@ -676,6 +697,9 @@ def create_pro_user_and_offerer(pro_user: ProUserCreationBodyModel) -> models.Us
 
     users_external.update_external_pro(new_pro_user.email)
 
+    action = history_api.log_action(history_models.ActionType.USER_CREATED, author=new_pro_user, user=new_pro_user)
+    objects_to_save.extend([action])
```

Comment:
il n'y en a qu'un seul à ajouter, tu peux simplifier en : 
`objects_to_save.append(action)`
et surtout... déplace ce bloc plus haut, AVANT `repository.save()`

Comme tu ne l'as pas détecté, il serait judicieux de vérifier dans un test auto qu'une action `USER_CREATED` est bien créée.

Reply:
Pourtant cette action est bien créée, j'ai réussi à la faire apparaître dans mon test produit...

Reply:
Ah oui pardon, elle est sauvegardée parce que tu n'as pas fait `save=False`.
Mais il est quand même mieux, avec cette option, de tout sauvegarder ensemble pour assurer la cohérence en un seul commit en base de données.

Reply:
Et il faut utiliser `[action]` sinon il cherche à itérer sur action lui-même et pas sur la liste des choses à sauvegarder.

Reply:
Avec `extend` mais pas avec `append`

Reply:
Au temps pour moi, je n'avais pas vu le append